pages/offices/[filename].tsx

```diff
@@ -189,7 +190,7 @@ const SidePanel = ({ office, testimonial }) => {
     <div className="prose max-w-full">
       <ContactPanel office={office} />
       <MicrosoftPanel />
-      <TestimonialPanel testimonial={testimonial} />
+      <TestimonialPanel props={testimonial} />
```

Comment:
```suggestion
      <TestimonialPanel {...testimonial} />
```

Reply:
Not done - I'm passing name as a string to this component